### ISSUE TITLE
Running awslocal in the latest image fails

### ISSUE THREAD

**Issue by georgijd** [localstack/localstack#1751]
Running the `awslocal` command in the `localstack/localstack:latest` image (digest `e022a3263089`) fails with

```
bash-5.0# awslocal
Traceback (most recent call last):
  File "/usr/bin/awslocal", line 29, in <module>
    from localstack_client import config
ModuleNotFoundError: No module named 'localstack_client'
```

Using the `localstack/localstack:0.10.5` image (digest `cbcdc058f1d2`) works:

```
bash-4.4# awslocal
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: too few arguments
```

**Comment by nickcaballero-mm** [comment]
Seeing the same issue. Added `pip install awscli-local` to top of script for the time being.

**Comment by xiang-zhu** [comment]
Probably related to https://github.com/localstack/localstack/issues/1753

**Comment by computermouth** [comment]
For those looking for an immediate fix, `localstack/localstack:0.10.4` works just fine
